eth_utils/address.py

```diff
@@ -56,20 +61,20 @@ def is_address(value):
         return False
 
 
-def to_normalized_address(address):
+def to_normalized_address(value: str) -> Address:
```

Comment:
This should take [`AnyStr`](https://docs.python.org/3/library/typing.html#typing.AnyStr) since it allows both `bytes` and `str` inputs.  Output type would be the `HexAddress` from my other comment.